quarkus.jdt.ext/pom.xml

```diff
@@ -33,7 +33,7 @@
     <repository>
       <id>jdt.ls.p2</id>
       <layout>p2</layout>
-      <url>https://download.eclipse.org/jdtls/milestones/1.13.0/repository/</url>
+      <url>https://download.eclipse.org/jdtls/milestones/1.14.0/repository/</url>
```

Comment:
If you are going to bump the referenced JDT-LS version in this PR, I think you should also update it on the qute-ls side (`qute.jdt/pom.xml`)

Reply:
Thanks. I missed that.